### PR TITLE
fix: README documentation improvements

### PR DESCRIPTION
**Pull request by guyzyl**
## Types of changes
Made several small improvements to `README.md`:
- Add `pip install` installation command line
- Small grammar improvements

- [ ] Hot fix (emergency fix and release)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation (change which affects the documentation site)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Release (`develop` -> `main`)

## Quality assurance (all should be covered).

- [x] My code follows the code style of this project.
- [x] Documentation for my change is up to date?
- [x] My PR meets testing requirements.
- [x] All new and existing tests passed.
- [x] All commits are signed-off.

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

# Before you merge

- Ensure it is a 'squash commit' if not a release.
- Ensure CI is currently passing
- Check sonar. If you are working for a fork a maintainer will reach out, if required.

### CODE REVIEW

**Review by fsuits — CHANGES_REQUESTED** [review]
Looks good but please resolve the linter issue.

And for "install and run" please point to the more complete link at https://ibm.github.io/compliance-trestle/python_trestle_setup/

**Review by enikonovad — CHANGES_REQUESTED** [review]
Thanks for this PR, looks good, but please merge the latest develop branch and run `make code-lint` and `make code-format` from the project root. Also please change the title to `fix: README documentation improvements`

Please refer to how to contribute to the project to here: `https://github.com/IBM/compliance-trestle/blob/develop/CONTRIBUTING.md`

**Comment by guyzyl** [comment]
@fsuits I added a reference to the full installation guide you linked.
@enikonovad merged latest `develop` branch and ran `make code-lint` + `make code-format` (although they install Python code formatters and I haven't committed any changes to Python code).

**Comment by deenine** [comment]
@guyzyl please use `make mdformat`

**Comment by guyzyl** [comment]
@deenine sure thing.
Ran it on `README.md` and everything looks good.

File: README.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ Users needing to import XML OSCAL artifacts are recommended to look at NIST's XM
 
 ## Python codebase, easy installation via pip
 
-Trestle runs on most all python platforms (e.g. Linux, Mac, Windows) and is available on PyPi so it is easily installed via pip.  It is under active development and new releases are made available regularly.
+Trestle runs on almost all Python platforms (e.g. Linux, Mac, Windows), is available on PyPi and can be easily installed via pip. It is under active development and new releases are made available regularly.\
```
Comment (deenine) [inline README.md:66]: Are the '\' s at the end of lines 66 and 67 intentional?

Comment (guyzyl) [reply README.md:66]: Yes, line breaks.

Comment (deenine) [reply README.md:66]: Use a newline instead?

Comment (guyzyl) [reply README.md:66]: The are in separate lines, but are rendered in a single line without a line break (`\`, `<br>`, or `&nbsp` )

**Review by enikonovad — APPROVED** [review]
LGTM

**Review by fsuits — APPROVED** [review]
LGTM

**Pull request merged**